src/app/features/characters/pages/character/character-details.component.html

```diff
@@ -1,12 +1,13 @@
-
 <div class="wrapper">
+  <app-banner></app-banner>
+
   <div class="character">
     <div class="picture-of-character">
-      <img src="{{character?.thumbnailUrl}}" alt="picture-of-character">
+      <img src="{{character.thumbnailUrl}}" alt="picture-of-character">
```

Comment:
why did you remove `?` ?

Reply:
We've already discussed this, I've added "?"